dbt_platform_helper/commands/copilot.py

```diff
@@ -249,8 +248,7 @@ def _get_s3_kms_alias_arns(session, application_name, config):
 def make_addons():
     """Generate addons CloudFormation for each environment."""
     output_dir = Path(".").absolute()
-    config_provider = ConfigProvider(ConfigValidator())
-    config_provider.config_file_check()
+    ConfigProvider.config_file_check()
```

Comment:
does this check belong in the `ConfigProvider`?
It feels like  a basic file check function which could be just a util